src/main/preload.js

```diff
@@ -130,6 +130,15 @@ contextBridge.exposeInMainWorld('electron', {
   toggleAutoLaunch() {
     return ipcRenderer.invoke('toggle-auto-launch');
   },
+  toggleDarkMode(mode) {
```

Comment:
toggleDarkMode as defined on preload.d.ts has to return a promise

Reply:
Right, fixing it, thanks!